extra/ide-overview.md

```diff
@@ -0,0 +1,55 @@
+# Markdown Files
```

Comment:
This file is very much the same as `extra/environment-setup.md`. I suppose it is some "copy+paste" mistake.